main.tf

```diff
@@ -107,7 +107,9 @@ resource "aws_lb" "main" {
 }
 
 resource "aws_lb_target_group" "https" {
-  name        = "ecs-${var.name}-${var.environment}-https"
+  # Name must be less than 32 characters, or AWS API returns error.
```

Comment:
This seems like good info to have in the docs for var.target_group_name.

Reply:
yep